cpp/dolfinx/fem/Form.h

```diff
@@ -77,6 +77,12 @@ class Form
   /// @param[in] mesh The mesh of the domain. This is required when
   /// there are not argument functions from which the mesh can be
   /// extracted, e.g. for functionals
+  /// @param[in] entity_maps If any coefficients in the form are not
```

Comment:
Would be nice to say first what `entity_maps` is, and then when it's needed.

I'm struggling bit with the explanation.